README.md

```diff
@@ -10,7 +10,8 @@ It lets you easily add real-time live chat and in-app messaging in your websites
 
 Kommunicate has a powerful human + chatbot hybrid platform, where you can integrate Dialogflow and Amazon Lex chatbot without using any code. You can also integrate any third-party or custom chatbot in Kommunciate as well. The automatic bot to human handoff enabled, customizable and rich-messaging equipped chat-widget gives you ample options to make your chatbot more powerful. 
 
-![Chat-widget](https://user-images.githubusercontent.com/32258004/95323958-9dd83c00-08bc-11eb-8744-bbf33140e00d.png)
+![Chat-widget](https://user-images.githubusercontent.com/91717856/215320134-f4f7308c-0e31-45cd-8c5b-619c4bba63b8.png)
```

Comment:
image is not good, it is showing restart thing, is that intentional?

Reply:
No no let me check once